_data/errors.yml

```diff
@@ -43,6 +43,16 @@ oidc:
         - Check the `acr_values` parameter in your authentication request.
         - Ensure that you are passing in the appropriate IAL Service Level value as well as any Authentication Assurance (AAL) values you might want for your application.
         More information about the `acr_values` parameter can be found here: [https://developers.login.gov/oidc/authorization/#acr_values](/oidc/authorization/#acr_values){:target="_blank"}
+    - title: No acceptable vots found
+      id: oidc-no-vot
+      accordion-id: no_vot
+      content: |
+        ##### Why it's happening
+        The vectors of trust found in the VTR parameter are not valid
+        ##### What to do:
+        - Check the `vot` parameter in your authentication request.
```

Comment:
Should this be `vtr`? In the Biometric Pilot documentation for partners, we are telling them to use the `vtr` parameter, such as `vtr=["Pb","P1"]`

Reply:
The oidc-sinatra sample app uses the `vtr` parameter in the auth request.